jib-plugins-common/src/test/java/com/google/cloud/tools/jib/plugins/common/logging/PlainConsoleLoggerTest.java

```diff
@@ -63,7 +63,7 @@ public void testLog() {
             Level.LIFECYCLE, Level.PROGRESS, Level.INFO, Level.DEBUG, Level.WARN, Level.ERROR),
         levels);
     Assert.assertEquals(
-        Arrays.asList("lifecycle", "progress", "info", "debug", "warn", "error"), messages);
+        Arrays.asList("lifecycle", " progress", "info", "debug", "warn", " error "), messages);
```

Comment:
> @vinod-tahelyani Just added a few comments:)
> 
> > @mpeddada1 we are expected to remove `\u001B[31;1m` kind as well right, in PlainConsoleLogger?
> 
> does this remove bright colors?

It will also remove a combination of (color + bold )